CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## [Unreleased]
 
 (Please put changes here)
+- Update Skeptic tests for Rusoto v0.41
```

Comment:
looks like this section is reserved for core changes that goes into next release. correct me if am wrong @matthewkmayer.

Reply:
It's generally used for bug fixes and new features, but putting this in is okay too. 😄